app/views/case_workers/admin/case_workers/index.html.haml

```diff
@@ -8,32 +8,32 @@
 
 = render partial: 'shared/search_form', locals: { search_path: case_workers_admin_case_workers_path(anchor: 'search-button'), hint_text: t('hint.search_caseworker'), button_text: t('search.caseworkers') }
 
-.table-container
-  %table.report
-    %caption
-      = t('.caption')
-    %thead
-      %th{ scope: 'col' }
-        = t('.surname')
-      %th{ scope: 'col' }
-        = t('.name')
-      %th{ scope: 'col' }
-        = t('.location')
-      %th{ scope: 'col' }
-        = t('.actions')
-    %tbody
-      - @case_workers.each do |case_worker|
-        %tr
-          %td
-            = case_worker.user.last_name
-          %td
-            = case_worker.user.first_name
-          %td
-            = case_worker.location.name
-          %td.user-controls
-            - if case_worker.active?
-              = govuk_link_to t('common.edit'), edit_case_workers_admin_case_worker_path(case_worker), 'aria-label': t('.edit_link_label', case_worker: case_worker.name)
-              = ' | '
-              = govuk_link_to t('common.delete'), case_workers_admin_case_worker_path(case_worker), method: :delete, data: { confirm: t('.confirmation') }, 'aria-label': t('.delete_link_label', case_worker: case_worker.name)
-            - else
-              Inactive
+
+.govuk-grid-row{class: 'govuk-!-margin-top-9'}
+  .govuk-grid-column-full
+
+    = govuk_table(class: 'report') do
```

Comment:
Do we still need to use this class ('report') ?

Reply:
@imtiazAhmd Yes, we still require it. It is used to make our table mobile friendly as the default govuk table classes do not.